packages/editor/src/components/inserter-menu-downloadable-blocks-panel/index.js

```diff
@@ -0,0 +1,44 @@
+/**
+ * External dependencies
+ */
+import { debounce } from 'lodash';
+
+/**
+ * WordPress dependencies
+ */
+import { __experimentalInserterMenuExtension } from '@wordpress/block-editor';
+import { DownloadableBlocksPanel } from '@wordpress/block-directory';
+import { useState } from '@wordpress/element';
+
+function InserterMenuDownloadableBlocksPanel() {
```

Comment:
@noisysocks and @talldan - it looks like all the code from '@wordpress/block-directory' will still get downloaded even when it's disabled for the user with the `GUTENBERG_PHASE` flag. We need to find a way to tree-shake this code for WordPress core.